Makefile

```diff
@@ -16,7 +16,7 @@ generate:
 test:
 	@echo "Running tests on:"; git symbolic-ref HEAD; git rev-parse HEAD
 	go test $(TEST) $(TESTARGS) -timeout=10s
-	@$(MAKE) vet
```

Comment:
You are right, also, the new version of make included in msys2 does work with $(MAKE) so we can close this. 
I was confused since AppVeyor did compile successfull even with the $(MAKE), and i used that to "verify" it would compile in non Windows systems.

In any case, this is resolved by using a compatible version of make, and one exists.